components/ledger/ledger-utxo-token-cache/src/main/kotlin/net/corda/ledger/utxo/token/cache/entities/TokenFilter.kt

```diff
@@ -0,0 +1,14 @@
+package net.corda.ledger.utxo.token.cache.entities
+
+interface TokenFilter: TokenEvent {
+
+    /**
+     * @return Returns the regular expression for a tag which the tokens will be matched against.
+     */
+    fun getTagRegex(): String?
```

Comment:
```suggestion
    val tagRegex: String?
```
IMO - it is a bit more Kotlin-native + if we do it that way - we can make implementation of this property in `data class` a bit more natural.